packages/client/src/components/Modal/Modal.tsx

```diff
@@ -6,45 +6,44 @@ import {
   StModalCloseIcon,
 } from './style';
 import { StButtonCloseModal } from 'components/Button/style';
+import { CSSProp } from 'styled-components';
 
 interface ModalType {
   children?: ReactNode;
   title?: string;
   handleCloseModal?: () => void;
-  width?: number;
-  verticalPaddings?: number;
-  horizontalPaddings?: number;
+  styles?: CSSProp;
+  isCloseOutside?: boolean;
+  isCrossButton?: boolean;
 }
 
 const Modal = (props: ModalType) => {
   const {
     title,
     handleCloseModal,
-    width,
-    verticalPaddings,
-    horizontalPaddings,
+    isCloseOutside,
+    isCrossButton,
```

Comment:
Что означает этот `isCrossButton`?

Reply:
кнопка крестика в правом верхнем углу  модалки, в каких-то модалках она будет, а в каких-то - нет

Reply:
Лучше наименовать так:
```suggestion
    hasCrossButton,
```